### ISSUE TITLE
[FEATURE REQUEST] Use scientific notation in global mass tables for extremely large or small values

### ISSUE THREAD

**Issue by msulprizio** [geoschem/gcpy#237]
## Add your feature request below:

In the global mass tables, extremely large values run off the line. For example:

```
###################################################################################
### Global mass (Gg) at end of simulation (Trop + Strat)                        ###
### Ref = GCC_14.0.0-rc.3; Dev = GCHP_14.0.0-rc.3                               ###
###################################################################################
                                    Ref                 Dev     Dev - Ref    % diff
N2                 : 3859772080128.000000  3859760847232.615234  -11232895.384766    -0.000
N2O                :     2479641.500000      2479907.104501    265.604501     0.011
N2O5               :         506.303284          505.198729     -1.104555    -0.218
NAP                :           0.000000            0.000000     -0.000000       nan
NH3                :         210.774109          214.981581      4.207472     1.996
NH4                :         402.096008          400.917130     -1.178878    -0.293
NIT                :         327.046692          357.625800     30.579108     9.350
NITs               :          57.952625           62.090785      4.138159     7.141
NO                 :         549.264465          551.113370      1.848905     0.337
NO2                :        1751.773804         1758.740821      6.967017     0.398
NO3                :          27.150970           26.819162     -0.331808    -1.222
NPHEN              :           1.051658            1.086208      0.034551     3.285
NPRNO3             :           8.573177            8.574375      0.001198     0.014
NRO2               :           0.000000            0.000000     -0.000000       nan
O                  :         150.155731          150.009617     -0.146115    -0.097
O1D                :           0.000026            0.000026      0.000000     0.020
O2                 : 1182735663104.000000  1182732344742.398926  -3318361.601074    -0.000
O3                 :     3198527.250000      3192510.018328  -6017.231672    -0.188
```

In addition, very low values can appear as zeroes which makes it difficult to distinguish very low values from actual zero values.

To avoid these two issues, we could use scientific notation when values exceed a certain number of digits.

**Comment by yantosca** [comment]
Thanks @msulprizio.  It looks like anything above 1e11 or below 1e-11 should be printed with scientific notation.  I'll work on this.

**Comment by yantosca** [comment]
We can close this issue as it has now been merged into the 1.4.0 development stream.
